data/event-schemas/schema/m.key.verification.m.relates_to.yaml

```
@@ -1,9 +1,9 @@
 ---
 description: |-
   Required when sent as an in-room message. Indicates the
-  `m.key.verification.request` that this message is related to. Note that for
-  encrypted messages, this property should be in the unencrypted portion of the
-  event.
+  [`m.key.verification.request`](#mkeyverificationrequest) that this message is
```

Given that this is specifically about in-room messages, this is linking to the wrong thing.

```suggestion
  [`m.key.verification.request`](#mroommessagemkeyverificationrequest) that this message is
```